src/System.Data.Extensions/dir.props

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\dir.props" />
+  <PropertyGroup>
+    <AssemblyVersion>4.1.1.0</AssemblyVersion>
+    <AssemblyKey>MSFT</AssemblyKey>
+    <IsNETCoreApp>true</IsNETCoreApp>
```

Comment:
Lets not set IsNETCoreApp or IsUAP properties here. That essentially makes them part of the shared framework for that platform and I'm not sure we want that yet.